src/components/HeaderSearch.js

```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components';
+
+const GreenHeader = styled.span`
```

Comment:
Why do you have two `GreenHeader` components? I would remove one of them and excluse the `GreenHeader` component into a separate file, export it and import where you need it to reduce repetitive code.

Reply:
The component isnt used anymore